packages/plugins/rubygem-smart_proxy_shellhooks/rubygem-smart_proxy_shellhooks.spec

```diff
@@ -0,0 +1,118 @@
+# template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name smart_proxy_shellhooks
+%global plugin_name shellhooks
+
+%global foreman_proxy_min_version 2.3
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.9.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Execute scripts via REST API from Foreman Webhooks plugin
+Group: Applications/Internet
+License: GPLv3
+URL: https://github.com/theforeman/%{gem_name}
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman-proxy >= %{foreman_proxy_min_version}
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-proxy-plugin-%{plugin_name} = %{version}
+# end specfile generated dependencies
+
+%{?scl:Obsoletes: rubygem-%{gem_name}}
```

Comment:
you can drop that, we never had a rubygm-shellhooks outside the scl.

Reply:
Dropped